app/assets/stylesheets/base/_fonts.scss

```diff
@@ -1,5 +1,5 @@
 // This file contains the base fonts for the application.
 
-@import url(https://fonts.googleapis.com/css?family=Lato:100,400,300);
+@import url(https://fonts.googleapis.com/css?family=Roboto:100,400,300);
 
-$font-family: 'Lato';
+$font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', sans-serif;
```

Comment:
@julianguyen I found out that Google Fonts uses version 1.0 of Lato (https://github.com/google/fonts/issues/6), with no certainty of when 2.0 will be uploaded. I've instead switched the CDN to jsdelivr which hosts version 2.0.